cognite/neat/rules/models/wrapped_entities.py

```diff
@@ -112,10 +129,35 @@ def from_dms_filter(cls, filter: dm.Filter) -> "DMSFilter":
                     if isinstance(entry, dict) and "space" in entry and "externalId" in entry
                 ]
             )
+        else:
+            return RawFilter(filter=json.dumps(dumped))
 
         raise ValueError(f"Cannot convert {filter._filter_name} to {cls.__name__}")
 
 
+class RawFilter(DMSFilter):
+    name: ClassVar[str] = "rawFilter"
+    inner: list[ContainerEntity] | None = None  # type: ignore[assignment]
```

Comment:
Overwriting the base class forcing this to None
```suggestion
    inner: None = None  # type: ignore[assignment]
```